src/Filters/molecules/DatePicker.tsx

```diff
@@ -359,22 +359,30 @@ const DatePicker: React.FC<IDatePicker> = ({
 
   const updateStartDate = useCallback((start: Date) => {
     const { end } = selectedRange ? selectedRange : TODAY_INTERVAL;
-    if (isAfter(add(start, { minutes: 1 }), end)) return;
+    if((isAfter(add(start, { minutes: 1 }), end))){
+      setCheckTimeValidation(true);
```

Comment:
Do we also need to return here if condition is true ? As previous version has return statement.

Reply:
Here we are showing red border for input box when this condition will TRUE. so no need to return.

Reply:
ok